resources/testharness.js

```diff
@@ -1795,20 +1802,25 @@
     /**
      * Assert that ``actual`` is a number greater than ``expected``.
      *
-     * @param {number} actual - Test value.
-     * @param {number} expected - Number that ``actual`` must be greater than.
+     * @param {number|bigint} actual - Test value.
+     * @param {number|bigint} expected - Value that ``actual`` must be greater than.
      * @param {string} [description] - Description of the condition being tested.
      */
     function assert_greater_than(actual, expected, description)
     {
         /*
-         * Test if a primitive number is greater than another
+         * Test if a primitive number (or bigint) is greater than another
          */
-        assert(typeof actual === "number",
+        assert(typeof actual === "number" || typeof actual === "bigint",
                "assert_greater_than", description,
                "expected a number but got a ${type_actual}",
                {type_actual:typeof actual});
 
+        assert(typeof actual === typeof expected,
+               "assert_less_than", description,
```

Comment:
This and following instances of "assert_less_than" need to be updated.

Reply:
Oops, thanks.

Reply:
Fixed in e7a3c470c2